SPECS/azurelinux-image-tools/azurelinux-image-tools.spec

```diff
@@ -75,14 +75,14 @@ install -p -m 0755 toolkit/out/tools/imagecustomizer %{buildroot}%{_bindir}/imag
 
 # Install container support files for imagecustomizer subpackage
 # These files are used when building the imagecustomizer container
-mkdir -p %{buildroot}/usr/local/bin
-mkdir -p %{buildroot}/
+mkdir -p %{buildroot}%{_bindir}
+mkdir -p %{buildroot}%{_libdir}/imagecustomizer
 
 # Copy container scripts from their source locations to container paths
-install -p -m 0755 toolkit/tools/imagecustomizer/container/entrypoint.sh %{buildroot}/usr/local/bin/imagecustomizer-entrypoint.sh
-install -p -m 0755 toolkit/tools/imagecustomizer/container/run.sh %{buildroot}/usr/local/bin/imagecustomizer-run.sh
-install -p -m 0755 toolkit/scripts/telemetry_hopper/telemetry_hopper.py %{buildroot}/usr/local/bin/telemetry_hopper.py
-install -p -m 0644 toolkit/scripts/telemetry_hopper/requirements.txt %{buildroot}/imagecustomizer-telemetry-requirements.txt
+install -p -m 0755 toolkit/tools/imagecustomizer/container/entrypoint.sh %{buildroot}%{_bindir}/imagecustomizer-entrypoint.sh
```

Comment:
Is there a reason these files wouldn't be under %{_libdir}/imagecustomizer as well? Components like `systemd` establish a precedent that binaries that are really internal to the component can be stored under an appropriate lib dir and not pollute the default PATH namespace.